spark/v3.1/spark/src/main/java/com/netease/arctic/spark/SparkSQLProperties.java

```diff
@@ -31,4 +31,8 @@ public class SparkSQLProperties {
   public static final String REFRESH_CATALOG_BEFORE_USAGE = "spark.sql.arctic.refresh-catalog-before-usage";
 
   public static final String REFRESH_CATALOG_BEFORE_USAGE_DEFAULT = "false";
+
+  public static final String CHECK_DATA_DUPLICATES_ENABLE = "spark.sql.check-data-duplicates.enabled";
```

Comment:
Does `spark.sql.arctic.check-source-data-uniqueness.enabled` property name is more appropriate？